x-pack/plugins/profiling/common/frame_group.ts

```diff
@@ -4,11 +4,15 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-
+import { last } from 'lodash';
 import { StackFrameMetadata } from './profiling';
 
 export type FrameGroupID = string;
 
+function stripLeadingSubdirs(sourceFileName: string) {
+  return last(sourceFileName.split('/'));
```

Comment:
Noob question: does it make sense to use pop() instead of importing a function from a 3rd party ?